packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -995,59 +1052,74 @@ export interface IBuildImage {
   runScriptBuildspec(entrypoint: string): BuildSpec;
 }
 
+interface LinuxBuildImageProps {
```

Comment:
Docstrings

Reply:
As this is a private interface that customer cannot use when constructing an image, do we need docstrings for it? The doc for it would be almost the same as the one in `IBuildImage`

Reply:
Done anyway.